rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/NetworkResult.scala

```diff
@@ -0,0 +1,362 @@
+    tx: Vector[RpcTransaction],
+    time: UInt32,
+    mediantime: UInt32,
+    nonce: UInt32,
+    bits: UInt32,
+    difficulty: BigDecimal,
+    chainwork: String,
+    previousblockhash: Option[DoubleSha256Digest],
+    nextblockhash: Option[DoubleSha256Digest])
+    extends NetworkResult
+
+case class RpcTransaction(
+    txid: DoubleSha256Digest,
+    hash: DoubleSha256Digest,
+    version: Int,
+    size: Int,
+    vsize: Int,
+    locktime: UInt32,
+    vin: Vector[TransactionInput],
+    vout: Vector[RpcTransactionOutput],
+    hex: Option[Transaction])
+    extends NetworkResult
+
+case class RpcTransactionOutput(
+    value: Bitcoins,
+    n: Int,
+    scriptPubKey: RpcScriptPubKey)
+    extends NetworkResult
+
+case class RpcScriptPubKey(
+    asm: String,
+    hex: String,
+    reqSigs: Option[Int],
+    scriptType: String,
+    addresses: Option[Vector[BitcoinAddress]])
+    extends NetworkResult
+
+case class ListSinceBlockResult(
+    transactions: Vector[Payment],
+    lastblock: DoubleSha256Digest)
+    extends NetworkResult
+
+case class Payment(
+    involvesWatchonly: Option[Boolean],
+    account: Option[String],
+    address: Option[BitcoinAddress],
+    category: String,
+    amount: Bitcoins,
+    vout: Int,
+    fee: Option[Bitcoins],
+    confirmations: Int,
+    generated: Option[Boolean],
+    blockhash: Option[DoubleSha256Digest],
+    blockindex: Option[Int],
+    blocktime: Option[UInt32],
+    txid: DoubleSha256Digest,
+    walletconflicts: Vector[DoubleSha256Digest],
+    time: UInt32,
+    timereceived: UInt32,
+    bip125_replaceable: String,
+    comment: Option[String],
+    to: Option[String])
+    extends NetworkResult
+
+case class ListTransactionsResult(
+    account: Option[String],
+    address: Option[BitcoinAddress],
+    category: String,
+    amount: Bitcoins,
+    label: Option[String],
+    vout: Option[Int],
+    fee: Option[Bitcoins],
+    confirmations: Option[Int],
+    trusted: Option[Boolean],
+    generated: Option[Boolean],
+    blockhash: Option[DoubleSha256Digest],
+    blockindex: Option[Int],
+    blocktime: Option[UInt32],
+    txid: Option[DoubleSha256Digest],
+    walletconflicts: Option[Vector[DoubleSha256Digest]],
+    time: UInt32,
+    timereceived: Option[UInt32],
+    comment: Option[String],
+    to: Option[String],
+    otheraccount: Option[String],
+    bip125_replaceable: String,
+    abandoned: Option[Boolean]
+) extends NetworkResult
+
+case class ReceivedAddress(
+    involvesWatchonly: Option[Boolean],
+    address: BitcoinAddress,
+    account: String,
+    amount: Bitcoins,
+    confirmations: Int,
+    label: String,
+    txids: Vector[DoubleSha256Digest])
+    extends NetworkResult
+
+case class GetTransactionResult(
+    amount: Bitcoins,
+    fee: Option[Bitcoins],
+    confirmations: Int,
+    generated: Option[Boolean],
+    blockhash: Option[DoubleSha256Digest],
+    blockindex: Option[Int],
+    blocktime: Option[UInt32],
+    txid: DoubleSha256Digest,
```

Comment:
This is sort of a nit, but let's use `txId` as that is what is used in the Transaction class in core. Another note, this is a big endian hash right?